validator_worker/src/heartbeat.rs

```diff
@@ -69,5 +60,5 @@ pub async fn heartbeat<A: Adapter + 'static>(
 }
 
 fn is_channel_not_exhausted(channel: &Channel, balances: &BalancesMap) -> bool {
```

Comment:
this is not very good, usually it's good to stick to the positive and shorter names, that's why in Rust you don't have `is_not_empty()` for example.